internal/auth/authutil/browser.go

```diff
@@ -62,3 +66,106 @@ func WaitForBrowserCallback(addr string) (code string, state string, err error)
 		return "", "", err
 	}
 }
+
+func resultPage(title string, message string, iconClass string) string {
```

Comment:
Ideally, we'd want this in an embedded HTML file, but for the time being (with the Friday launch near) this can do.